### ISSUE TITLE
Aure Pipelines Executing Old Tests

### ISSUE THREAD

**Issue by seanlaw** [TDAmeritrade/stumpy#121]
I've noticed a lot of inconsistencies in Azure Pipelines lately. Specifically, [in this one failed test case](https://dev.azure.com/stumpy-dev/stumpy/_build/results?buildId=358), the test is failing with:

```
================================== FAILURES ===================================
________________ test_two_constant_subsequences_A_B_join_swap _________________

dask_client = <Client: 'tcp://127.0.0.1:1808' processes=1 threads=2, memory=7.52 GB>

    @pytest.mark.filterwarnings("ignore:\\s+A large number of values are smaller")
    @pytest.mark.filterwarnings("ignore:\\s+For a self-join")
    @pytest.mark.filterwarnings("ignore:numpy.dtype size changed")
    @pytest.mark.filterwarnings("ignore:numpy.ufunc size changed")
    @pytest.mark.filterwarnings("ignore:numpy.ndarray size changed")
    @pytest.mark.filterwarnings("ignore:\\s+Port 8787 is already in use:UserWarning")
    def test_two_constant_subsequences_A_B_join_swap(dask_client):
        T_A = np.array([0, 0, 0, 0, 0, 1, 1, 1, 1, 1], dtype=np.float64)
        T_B = np.concatenate((np.zeros(20, dtype=np.float64), np.ones(5, dtype=np.float64)))
        m = 3
        left = np.array(
            [utils.naive_mass(Q, T_B, m) for Q in core.rolling_window(T_A, m)], dtype=object
        )
>       right = stumped(dask_client, T_B, m, T_A, ignore_trivial=False)

tests\test_stumped_constant_subsequences.py:216: 
```

But what is really strange is that this test doesn't exist in the latest commit anymore! That is:

1. The test file `tests\test_stumped_constant_subsequences.py` doesn't exist (it was moved to a different test file name a few commits ago)
2. The real test file that contains this test function `test_two_constant_subsequences_A_B_join_swap` doesn't contain this line `T_A = np.array([0, 0, 0, 0, 0, 1, 1, 1, 1, 1], dtype=np.float64)`

In other words, even though Azure Pipelines claims to be retrieving the latest version of the code from the Github repository, it is somehow still loading/referencing old files from an old commit. I need help figuring out what the problem is.

**Comment by seanlaw** [comment]
@cuichenli Any chance you have time to help me figure out what is going on here? Or where I might be able to find help?

**Comment by seanlaw** [comment]
Nvm, I was able to resolve this
